consensus/pacemaker/pacemaker.go

```diff
@@ -221,16 +235,9 @@ func (p *paceMaker) NewHeight() {
 func (p *paceMaker) startNextView(qc *typesCons.QuorumCertificate, forceNextView bool) {
 	// DISCUSS: Should we lock the consensus module here?
 
-	p.consensusMod.step = NewRound
-	p.consensusMod.clearLeader()
-	p.consensusMod.clearMessagesPool()
-	// TECHDEBT: This should be avoidable altogether
-	if p.consensusMod.utilityContext != nil {
-		if err := p.consensusMod.utilityContext.Release(); err != nil {
-			log.Println("[WARN] Failed to release utility context: ", err)
-		}
-		p.consensusMod.utilityContext = nil
-	}
+	p.GetBus().GetConsensusModule().SetStep(uint64(NewRound))
```

Comment:
we can create a local variable called `consensusMod := p.GetBus().GetConsensusModule()` to simplify the code a bit. DItto elsewhere.

Wdyt?

Reply:
agreed, done.